src/Fmt_ast.ml

```diff
@@ -1704,8 +1706,8 @@ and fmt_expression c ?(box = true) ?pro ?epi ?eol ?parens ?(indent_wrap = 0)
                $ fmt "@ " $ fmt_cases c ctx'' cs $ str ")"
                $ Cmts.fmt_after c pexp_loc $ fmt_atrs ))
       | _ ->
-          wrap_if parens "(" ")"
-            (hvbox 2 (fmt_args_grouped e0 e1N1) $ fmt_atrs) )
+          wrap_if ~box:(hvbox 2) parens "(" ")"
+            (fmt_args_grouped e0 e1N1 $ fmt_atrs) )
```

Comment:
The box would also contain the attributes, we need some tests to check how different it would look.

Reply:
Should works ok. However, while doing some testing I noticed that this fix does not work with `wrap-fun-args=true`.